plugins/modules/zos_volume_init.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) IBM Corporation 2022
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: zos_volume_init
+short_description: Initialize volumes or minidisks on z/OS through ICKDSF init command.
+description:
+  - Wraps the ICKDSF INIT command to initialize volume or minidisk.
+  - Writes volume label and VTOC on volume or minidisk.
+version_added: 1.6.0
+author:
+  - "Austen Stewart (@stewartad)"
+  - "Almigdad Suliman (@Almigdad-Suliman)"
+  - "Nicholas Teves (@TODO)"
+  - "Nuoya Xie (@nxie13)"
+  - "Trevor Glassey (@tkglassey)"
+  - "Tyler Edwards (@TLEdwards-Git)"
+  - "Ketan Kelkar (@ketankelkar)"
+
+options:
+  volume_address:
+    description:
+      - 3 or 4 hexadecimal digit address of the volume to initialize.
+      - This could be a unit address or device number.
+    required: true
+    type: str
+  verify_existing_volid:
+    description:
+      - Verify that the provided volume serial matches the one found on existing volume/minidisk.
```

Comment:
I like the line below I had included because the parameter seems to be a boolean (when i first picked it up, i assumed bool too)

"Note - This option is NOT a boolean, please leave it blank in order to skip the verification."

Also the other params `verify_*` are bools, which in my mind adds to the ambiguity and therefore warrants the Note